rag_experiment_accelerator/config/paths.py

```diff
@@ -13,13 +16,34 @@ def get_all_file_paths(directory: str) -> list[str]:
     return [file for file in glob.glob(pattern, recursive=True) if os.path.isfile(file)]
 
 
+def try_create_directory(directory: str) -> None:
+    """
+    Tries to create a directory with the given path.
+
+    Args:
+        directory (str): The path of the directory to be created.
+
+    Returns:
+        None
+
+    Raises:
+        OSError: If an error occurs while creating the directory.
+    """
+    try:
+        os.makedirs(directory, exist_ok=True)
+    except OSError as e:
+        if "Read-only file system" in e.strerror:
```

Comment:
What this if does if there is "pass" inside it? It can be omitted

But in general - what this try-except does there? This is a very specific case to have a separate function for that. I don't mean that it should be removed (if should be tho), just curious

Reply:
I only changed the function name. Do not know the whole story behind it.